### ISSUE TITLE
Segmentation faults since interpret 0.1.11

### ISSUE THREAD

**Issue by martinsotir** [microsoft/interpret#46]
Hi, thank you for your great work. 

I just tried updating from 0.1.10 to 0.1.11 using pip and I am getting segmentation faults.
The issue does not seem to originate from a memory resource limitation (I have a 10 GB free memory margin when using 0.1.10) .
I was not able to diagnose further.

Platform : x86_64, Linux, ubuntu 16

Code extract: (works fine with 0.1.10)

```python
_ebm = ExplainableBoostingRegressor(
    n_estimators=16,
    learning_rate=0.1,
    early_stopping_run_length=10,
    data_n_episodes=200,
    n_jobs=1, # same error with n_jobs=-2
    feature_names=_categorical_features + _numerical_features,
    feature_types= ['categorical'] * len(_categorical_features) + ['continuous'] * len(_numerical_features))

_ebm.fit(X, y)  # X: 108,728 x 39 numpy array
```

GDB trace:

```bash
# Segmentation fault (core dumped):

Thread 1 "python" received signal SIGSEGV, Segmentation fault.
__GI___libc_free (mem=0x2) at malloc.c:2951
2951    malloc.c: No such file or directory.
(gdb) bt
#0  __GI___libc_free (mem=0x2) at malloc.c:2951
#1  0x00007ffed376128c in DataSetAttributeCombination::~DataSetAttributeCombination() ()
   from /home/ubuntu/miniconda3/envs/env/lib/python3.7/site-packages/interpret/glassbox/ebm/../../lib/lib_ebmcore_linux_x64.so
#2  0x00007ffed377508d in FreeTraining ()
   from /home/ubuntu/miniconda3/envs/env/lib/python3.7/site-packages/interpret/glassbox/ebm/../../lib/lib_ebmcore_linux_x64.so
```

Python logs tail: (lots of INFO and ERROR messages since this release !).

```
[...]
INFO: Entered GetBestModel: ebmTraining=0x555e1f55ad90, indexAttributeCombination=38
INFO: Exited GetBestModel 0x555e24b7e6f0
INFO: Deallocation start
INFO: Entered FreeTraining: ebmTraining=0x555e1f55ad90
INFO: Entered ~EbmTrainingState
INFO: ~EbmTrainingState identified as regression type
INFO: Entered ~CachedTrainingThreadResources
INFO: Exited ~CachedTrainingThreadResources
INFO: Entered SamplingWithReplacement::FreeSamplingSets
INFO: Entered ~SamplingWithReplacement
INFO: Exited ~SamplingWithReplacement
INFO: Exited SamplingWithReplacement::FreeSamplingSets
INFO: Entered ~DataSetAttributeCombination

Segmentation fault (core dumped)
```

(Full log here: [ebm_error_log.txt](https://github.com/microsoft/interpret/files/3488276/ebm_error_log.txt) ).

**Comment by interpret-ml** [comment]
Hi @martinsotir -

Thank you for reporting this!  We believe we've found and fixed the issue, and are in the process of putting out a new release to address this.  This problem only manifests itself if you have features that contain a single value.

- InterpretML Team

**Comment by interpret-ml** [comment]
Hi @martinsotir -

The fix is released now as version 0.1.12.  Please try it out and let us know if everything is working for you now!

-InterpretML Team

**Comment by martinsotir** [comment]
Thanks for the quick answer, dropping single-valued features did the trick.
I can confirm that the 0.1.12 release also fix this issue!
